sos/report/plugins/aap_controller.py

```diff
@@ -0,0 +1,96 @@
+# Copyright (c) 2023 Pavel Moravec <pmoravec@redhat.com>
```

Comment:
Copy error?

Reply:
I can leave with it.. :)

In fact, the plugin was internally prepared in a collaborative manner by few fellows, also with my contribution. I would vote for putting here my and Lucas's name, but I am ok with either option.

Reply:
@pmoravec++ made the initial draft and i just adjusted some points